Calculator/Program.cs

```diff
@@ -48,13 +49,12 @@ static void Main(string[] args)
                     }
                     mathOperations.MathOperation();
                 }
-                catch(FormatException)
+                else
                 {
-                    Console.WriteLine("No option selected or wrong character entered.");
+                    Console.WriteLine("No option selected or wrong character entered");
```

Comment:
Can `SwitchDefault()` be used here? Of course it should be renamed, because `SwitchDefault()` doesn't switch anything, it just prints something to the console.